src/world.rs

```diff
@@ -36,6 +36,15 @@ use crate::{
 ///
 /// The components of entities who have the same set of component types are stored in contiguous
 /// runs, allowing for extremely fast, cache-friendly iteration.
+///
+/// There is a max number of unique entity IDs, which means that there is a max number of live
+/// entities. When old entities are despawned, their IDs will be reused on a future entity, and
+/// old entity handles with that ID will be invalidated.
+///
+/// If an ID is reused enough times (around 4 billion times), previously dead entity handles
+/// will erroneously become live again.  The likeliness of this issue occurring is pretty rare
+/// as an ID needs to be reused enough times AND an old handle needs to be kept around in the
+/// mean time.
```

Comment:
> previously dead entity handles will erroneously become live again

This reads as if *all* previously dead handles will simultaneously become live, which is incorrect.

> The likeliness of this issue occurring is pretty rare

Could be phrased more concisely as "This is unlikely to occur" or similar.

> ID

The notion of "ID" used here and above may not be very clear to the reader, since we use that term to refer to `Entity` values elsewhere. Maybe just describe in more broad terms that after billions of entities have been despawned (specifically) it is possible for a complete `Entity` value to be reused, which may collide with any extremely long-lived references to dead entities?

Reply:
That sound good.